sourmash/sbt.py

```diff
@@ -254,7 +263,11 @@ def find(self, search_fn, *args, **kwargs):
 
                     # leaf node? it's a match!
                     if isinstance(node_g, Leaf):
-                        matches.append(node_g)
+                        # Return tuple of node data and position
+                        if return_pos:
+                            matches.append((node_g, node_p))
```

Comment:
I think a named tuple/ dataclass here would be nice, especially because I could see how someone could easily confuse `node_g` and `node_p` variables, but it could be overkill depending on how often `matches` is used by the end user: 

maybe something like 
`MatchedNode(data=node_g, position=node_p)`